client/styles/styles.css

```diff
@@ -472,3 +472,202 @@ input[type="password"] {
   margin-bottom: 1em;
   display: flex;
 }
+
+/*About-us Section*/
+
+.about-us-container {
+  font-size: 24px;
+  background-color: #DFD7EF;
+  color: #1C2294;
+  height: 120px;
+  font-weight: 200;
+  font-family: 'poppins';
+}
+.about-us-container h1{
+  position: absolute;
+  top: 16%;
+  left: 50%;
+  transform: translateX(-50%) translateY(-50%);
+}
+.About-hyf-team {
+margin-left: 47.5%;
+margin-top: -15em;
+}
+
+
+.About-hyf {
+  background-color: #fff;
+  border: 2px solid  #1C2294;
+  padding: 50px;
+  margin: 20px;
+  margin-left: 10%;
+  font-size: 20px;
+  margin-top: 5%;
+  font-style: 'Poppins';
+  width: 600px;
+  width: min(600px,70%);
+}
+
+.About-visit-website{
+padding: 14px 26px;
+background-color: #2F195F;
+border-radius: 20px;
+color: #fff;
+font-weight: 600;
+font-size: 14px;     
+text-decoration: none; 
+}
+
+.About-hyf-connect {
+background-color: #fff;
+border: 2px solid #1C2294;
+margin-top: 2em;
+font-size: 20PX;
+width: 600px;
+padding: 50px;
+margin-left:35em;
+
+}
+.About-visit-project{
+padding: 14px 26px;
+background-color: #2F195F;
+border-radius: 20px;
+color: #fff;
+font-weight: 600;
+font-size: 14px;
+float: right; 
+margin-top: -1em;
+text-decoration: none;
+}
+
+.About-hyf-connect2 {
+  margin-top: -35em;
+}
+.team-container{
+  font-size: 24px;
+  color: #1C2294;
+  text-align: center;
+}
+.About-title-1 {
+  color: #1C2294;
+  text-align: right;
+  text-shadow: #2F195F;
+  font-size: 24px;
+  margin-top: 5em;
+  font-weight: 600;
+  margin-right: 5em;
+
+}
+.About-title {
+  color: #1C2294;
+  text-shadow: #2F195F;
+  font-size: 30px;
+  font-weight: 600;
+  box-shadow: 0 0 22px #3336;
+  border-radius: 3%;
+  padding: 20px 24px;
+  width: 500px;
+  height: 70px;
+  margin: 0 auto;
+  margin-top: 1em;
+
+
+}
+
+.caption h3{
+  color: #1C2294;
+  text-align: center;
+  text-shadow: #2F195F;
+  font-size: 2em;
+  margin-top: 1em;
+  font-weight: 600;
+}
+
+.About-main{
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  background-color: #fdfdfd;
+  flex-wrap: wrap;
+  margin-top: 15em;
+  box-sizing: border-box;
+  
+  
+  }
+
+.profile-card{
+  position: relative;
+  font-family: sans-serif;
+  width: 220px;
+  height: 220px;
+  background: #fff;
+  padding: 30px;
+  border-radius: 50%;
+  box-shadow: 0 0 22px #3336;
+  transition: .6s;
+  margin: 0 25px;
+  margin-top: -30em;
+  margin-left: 3em;
+  display: grid;
+  
+  
+}
+
+.profile-card:hover{
+  border-radius: 10px;
+  height: 260px;
+}
+
+.profile-card .img{
+  position: relative;
+  width: 100%;
+  height: 100%;
+  transition: .6s;
+  z-index: 99;
+
+}
+
+.img img{
```

Comment:
Please add about- prefix to the class name something like about-img

And... as a quick fix for the 'ovals'


```
.img img{
  width: 100%;
  min-width: 20%;
  border-radius: 50%;
  box-shadow: 0 0 22px #3336;
  transition: .6s;  
}
```

Reply:
ok, I will fix them

Reply:
I made a mistake.
It needs to be 

```
.about-img img{
  width: 160px;
  height: 160px;
  object-fit: cover;
  border-radius: 50%;
  box-shadow: 0 0 22px #3336;
  transition: .6s; 
}
```